shared/chat/inbox/row/chat-inbox-header/index.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import * as React from 'react'
+import {isDarwin} from '../../../../constants/platform'
+import ChatFilterRow from '../chat-filter-row'
+import StartNewChat from '../start-new-chat'
+
+type Props = {
+  filter: string,
+  filterFocusCount: number,
+  isLoading: boolean,
+  onHotkey: (cmd: string) => void,
+  onNewChat: () => void,
+  onSelectDown: () => void,
+  onSelectUp: () => void,
+  onSetFilter: (filter: string) => void,
+  showNewChat: boolean,
+}
+
+const ChatInboxHeader = (props: Props) =>
```

Comment:
the other option is to have 3 connectors here, one for each component, and this one which just figures out showNewChat, thats likely cleaner